cookbooks/aws-parallelcluster-environment/recipes/init/config_default_user_home.rb

```diff
@@ -14,6 +14,20 @@
 
 return if node['cluster']['default_user_home'] == 'shared'
 
+# Stop sshd and close all connections
+service 'sshd' do
+  action :stop
+  sensitive true
+end
+bash "Close ssh connections to perform a default user move" do
```

Comment:
We should mention in the changelog that we close SSH connection during the bootstrap of the node when the user home is moved. I'm suggesting to mention that because, testers and SAs (IDK if also CX) are used to ssh into the head node immediately and they would get confused by sudden close of sessions.